app/views/theblog/content_nodes/show.html.haml

```diff
@@ -26,3 +26,23 @@
           %ul
             - @node.child_nodes.each do |child|
               %li= link_to_content_node(child)
+
+        - if @node.is_a? Theblog::Post
+          %h4 Comments
+
+          - @node.comments.each do |comment|
+            .comment
+              %blockquote
+                = comment.title
+                %small
+                  = comment.author.first_name
+                  = comment.author.last_name
+
+          - if current_account
+            %h4 Add your comment
```

Comment:
BTW.
There is the task for this but I agree
If you can It would be great if you made your changes using `I18n`